docs/FAQ.md

```diff
@@ -37,6 +37,34 @@ Breaking large amounts of data up into manageable groups is generally a good ide
 
 No. VIAME Web and DIVE Desktop perform automatic transcoding if it is necessary.
 
+## How does video frame alignment work?
+
+When you annotate a video in DIVE, the true video is played in the browser using a native HTML5 video player.
+
+Web browsers report and control time in floating point seconds rather than using frame numbers, but annotations are created using frame numbers as their time indicators, so it's important to make sure these line up.
+
+Most of the time, videos are **downsampled** for annotation, meaning that the true video framerate (30hz, for example) is annotated at a lower rate, such as 5hz or 10hz.  Kwiver (the computer vision tool behind VIAME) uses a downsampling approach that sorts actual frames into downsampled buckets based on the start time of the frame.
+
```

Comment:
Maybe just reiterate that if possible to use a whole number divisor of the frame rate.